taxi_service/urls.py

```diff
@@ -22,4 +22,5 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("registration/", include("django.contrib.auth.urls")),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
The use of `static()` is appropriate for serving static files during development. However, remember to configure a proper static files server for production, as `static()` is not suitable for production use.